app/routes/core.rb

```diff
@@ -21,7 +21,7 @@ class Core < Sinatra::Application
         enable :show_exceptions
       end
 
-      unless settings.test?
+      unless settings.test? || settings.development?
```

Comment:
Could be something strange on my end. I was definitely getting the 404 page instead of the exceptions until I commented that out.

Do you all have a Bugsnag API key locally?